src/OpenFin/openfin.ts

```diff
@@ -258,11 +293,8 @@ export class OpenFinContainer extends WebContainerBase {
         return this.wrapWindow(new this.desktop.Window(newOptions));
     }
 
-    public showNotification(options: NotificationOptions) {
-        const msg = new this.desktop.Notification({
-            url: options.url,
-            message: { message: options.message }
-        });
+    public showNotification(title: string, options?: NotificationOptions) {
+        const msg = new this.desktop.Notification(ObjectTransform.transformProperties(options, this.notificationOptionsMap));
```

Comment:
Since NotificationOptions standardized on "body" for the message to match W3C notification options, we need to copy "body" to "message" to provide as options to OpenFin.